README.md

```diff
@@ -1,6 +1,7 @@
 # Looper
 
 [![Documentation Status](http://readthedocs.org/projects/looper/badge/?version=latest)](http://looper.readthedocs.io/en/latest/?badge=latest)
+[![Build Status](https://travis-ci.org/vreuter/looper.svg?branch=features_0.5)](https://travis-ci.org/vreuter/looper)
```

Comment:
Yeah, I pointed it to `master` now. Open to `dev` instead (or probably both is easy, too), if we want to display status for our active development instead/also. This is on my fork, though, not `epigen`, since I'm not sure if it's OK to allow Travis to access this code in `epigen`, and I don't think that I have sufficient permissions to grant it access if we do want to do that.